prow/cluster/deck_deployment.yaml

```diff
@@ -43,14 +43,27 @@ spec:
         - --tide-url=http://tide/
         - --hook-url=http://hook:8888/plugin-help
         - --redirect-http-to=prow.k8s.io
+        - --oauth-url=/user-dashboard
         volumeMounts:
+        - name: oauth-config
+          mountPath: /ect/github
+          readOnly: true
+        - name: cookie
```

Comment:
the volume and volumeMount names must match: https://github.com/kubernetes/test-infra/pull/6830

Reply:
fix here https://github.com/kubernetes/test-infra/pull/6832